src/components/AddressInput/AddressInput.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+import {bool, oneOf, ValidationMap} from 'prop-types';
+import {AddressInput as CoreAddressInput, AddressInputProps as CoreAddressInputProps} from 'wix-ui-core/AddressInput';
+import style from './AddressInput.st.css';
+import Location from 'wix-ui-icons-common/Location';
+import omit = require('lodash/omit');
+import {Omit} from '../../types/common';
+import Search from 'wix-ui-icons-common/Search';
+
+const THROTTLE_INTERVAL = 150;
+
+const excludePropsArray = ['forceContentElementVisibility', 'forceOptions', 'throttleInterval', 'locationIcon'];
+export type excludeProps = 'forceContentElementVisibility' | 'forceOptions' | 'throttleInterval' | 'locationIcon';
```

Comment:
To prevent this duplication, we have a nice util function `enumValues` or something in the util file.
Check it out maybe it can help you here.

Reply:
The reason that it's duplicated is because union types cannot be transformed into array